packages/roosterjs-editor-plugins/lib/plugins/WatermarkV2/WatermarkV2.ts

```diff
@@ -0,0 +1,119 @@
+import {
+    EditorPlugin,
+    Entity,
+    IEditor,
+    PluginEvent,
+    PluginEventType,
+} from 'roosterjs-editor-types';
+import { kebabCase } from './utils';
+
+const ENTITY_TYPE = 'WATERMARK_WRAPPER';
+const WATERMARK_ATTR = 'data-rooster-watermark';
+const STYLE_ID = 'watermarkStyle';
```

Comment:
There can be issue if there are more than 1 editor instances in the same page and one of them is disposed, then this style tag will be removed.

Reply:
Maybe use [ensureUniqueId](https://github.com/microsoft/roosterjs/blob/2e598554cbd848ebb1fcd823656f7aa4d0aa47b2/packages/roosterjs-editor-core/lib/coreApi/selectTable.ts#L195)? Can we make this exported?